compute/cker/include/cker/operation/MatrixBandPart.h

```diff
@@ -32,10 +32,10 @@ void MatrixBandPart(const T num_lower_diags, const T num_upper_diags, const Shap
 {
   auto last_dim = input_shape.DimensionsCount() - 1;
 
-  T batch_num = 0;
+  T batch_num = 1;
   for (int dim = 0; dim < last_dim - 2; dim++)
```

Comment:
`last_dim - 2` should also be changed to `input_shape.DimensionsCount() - 2`. If I'm right, could you change it too?